pom.xml

```diff
@@ -13,7 +13,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <kc.version>5.2.1</kc.version>
-        <rice.version>2.3.7</rice.version>
+        <rice.version>2.3.9</rice.version>
         <servlet.version>2.5</servlet.version>
```

Comment:
Yup, this was just temporary to be able to build the prototype build  and deploy it on DEV. This commit will be taken out of the final version.